grading/main_runner.cpp

```diff
@@ -98,48 +110,101 @@ int main(int argc, char *argv[]) {
       continue;
     }
 
-    std::cout << "========================================================" << std::endl;
-    std::cout << "TEST #" << i << std::endl;
+    if ( generation_type == "output" ) {
```

Comment:
The clauses of this if statement are nearly duplicates. Can we either break the duplicated portion into the function or else find a way to reduce lines of nearly duplicated code?

Reply:
Added header function

Reply:
Can you briefly explain what is the issue?

Reply:
@emaicus  Can you please review it again